### PR TITLE
Formats json using ?pretty

### PR DESCRIPTION
**Pull request by shimastripe**
## WHY

#60 

## WHAT

- if `?pretty==true`, formats JSON output.

### Example

- `:8080/users?pretty`

### CODE REVIEW

File: _templates/controller.go.tmpl [hunk 1]
```diff
@@ -61,7 +61,12 @@ func Get{{ pluralize .Model.Name }}(c *gin.Context) {
 	for _, {{ tolower .Model.Name }} := range {{ pluralize (tolower .Model.Name) }} {
 		fieldMap = append(fieldMap, helper.FieldToMap({{ tolower .Model.Name }}, fields))
 	}
-	c.JSON(200, fieldMap)
+	_, ok := c.GetQuery("pretty")
```
Comment (dtan4) [inline _templates/controller.go.tmpl]: Please insert empty line above.

File: _templates/controller.go.tmpl [hunk 2]
```diff
@@ -91,7 +96,12 @@ func Get{{ .Model.Name }}(c *gin.Context) {
 	}
 
 	fieldMap := helper.FieldToMap({{ tolower .Model.Name }}, fields)
-	c.JSON(200, fieldMap)
+	_, ok := c.GetQuery("pretty")
```
Comment (dtan4) [inline _templates/controller.go.tmpl]: Please insert empty line above.

**Comment by shimastripe** [comment]
@dtan4 Sorry, I fixed all files.

**Comment by dtan4** [comment]
I'm sorry but one more thing: write explanation of `pretty=` parameter in README API server specification, please.

**Comment by shimastripe** [comment]
Update `readme.md`!
Thank you for writing the other specifications.

File: README.md [hunk 3]
```diff
@@ -194,6 +194,7 @@ Resource name is written in the plural form.
 |---------|-----------|-------|-------|
 |`fields=`|Fields to receive|All fields|`name,emails.address`|
 |`preloads=`|Nested resources to preload|(empty)|`emails,profile`|
+|`pretty`|Formats JSON output|`false`|Not required value|
```
Comment (dtan4) [inline README.md]: Hmm...

- Description: Prettify JSON response
- Example: `true`

would be better.

**Comment by shimastripe** [comment]
@dtan4 Okay! Fixed.

**Comment by dtan4** [comment]
LGTM 👍 Thanks ❗

**Pull request merged**